lib/kafo/kafo_configure.rb

```diff
@@ -166,6 +166,12 @@ def logger
     def run(*args)
       started_at = Time.now
       logger.debug("Running installer with args #{args.inspect}")
+      if config.app[:verbose]
+        log_output_message = <<~MESSAGE.strip
+          Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}. Use -l to change output verbosity to one of ERROR, WARN, NOTICE, INFO, or DEBUG.
```

Comment:
Small nit: first you speak of `level X` and then about `output verbosity`. Would it make sense to always speak of level? I also think log levels are a well known concept any admin should understand while `output verbosity` may not. Something like
```suggestion
          Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}. Use -l to set the level to ERROR, WARN, NOTICE, INFO, or DEBUG.
```

I'd also consider adding a newline after the dot to stay somewhat close to the max line length.

Reply:
Done. Thanks for the suggestion!